torchtune/utils/_checkpointing/_checkpointer.py

```diff
@@ -264,6 +267,10 @@ def save_checkpoint(
                 f"{os.path.getsize(output_path) / 1000**3:.2f} GB "
                 f"saved to {output_path}"
             )
+        elif adapter_only:
```

Comment:
This logic doesn't make sense to me. Why wouldn't this go above?

Reply:
This branch gets called if there wasn't an adapter key in the config (the first if is false) but adapter_only is true